src/blob_file_size_collector.cc

```
@@ -72,10 +72,9 @@ Status BlobFileSizeCollector::Finish(UserCollectedProperties* properties) {
   }
 
   std::string res;
-  if (!Encode(blob_files_size_, &res) || res.empty()) {
-    fprintf(stderr, "blob file size collector encode failed\n");
-    abort();
-  }
+  bool ok __attribute__((__unused__)) = Encode(blob_files_size_, &res);
+  assert(ok);
```

we may ignore the error in release mode?